core/blockchain/impl/cached_tree.hpp

```diff
@@ -164,15 +81,15 @@ namespace kagome::blockchain {
      */
     void removeFromMeta(const std::shared_ptr<TreeNode> &node);
```

Comment:
We still have a meta-data? It seems as removed

Reply:
Still used, will be removed in other pr.